src/Microsoft.Identity.Web/AppServicesAuth/AppServicesAuthenticationInformation.cs

```diff
@@ -190,7 +189,7 @@ public static bool IsAppServicesAadAuthenticationEnabled
                     jsonWebToken.Claims,
                     idp,
                     isAadV1Token ? Constants.NameClaim : Constants.PreferredUserName,
-                    ClaimsIdentity.DefaultRoleClaimType));
+                    "roles"));
```

Comment:
nit: We have a constant in TokenAcquisition for [Roles](https://github.com/AzureAD/microsoft-identity-web/blob/c1135ea1831443182304ad31aeb3562c67e72dfe/src/Microsoft.Identity.Web.TokenAcquisition/ClaimConstants.cs#L71)

Reply:
Thanks. Fixed